.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+
+branches:
+  only:
+  - develop 
```

Comment:
I noticed that when @tonyyang-svail configured Travis CI for sqlflow, he run CI on all branches

https://github.com/sql-machine-learning/sqlflow/blob/58cd71930b576ab95f8abc9b00536e71cfed3120/.travis.yml#L3-L6

but only deploys the develop branch

https://github.com/sql-machine-learning/sqlflow/blob/58cd71930b576ab95f8abc9b00536e71cfed3120/.travis.yml#L32-L37

Does the configuration here mean that we only do CI when the develop branch is changed? I think CI means testing any feature branch before we merge it into the develop branch, right? If so, it seems that we should CI all branches here?